types/yaireo__relative-time/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for @yaireo/relative-time 1.0
+// Project: https://github.com/yairEO/relative-time
+// Definitions by: Ben Grynhaus <https://github.com/bengry>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Settings {
+    /**
+     * @default browser locale
+     * @example 'es'
+     */
+    locale?: string;
+
+    /**
+     * @default { numeric: 'auto' }
+     */
+    options?: Intl.DateTimeFormatOptions;
+}
+
+declare class RelativeTime {
+    constructor(settings?: Settings);
+
+    from(d1: Date, d2?: Date): string;
+}
+
+export = RelativeTime;
```

Comment:
```suggestion
export = RelativeTime;

export as namespace RelativeTime
```

This is UMD like library IMO:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library

Reply:
You're right. I forgot to include that. I accepted your suggestion